### ISSUE TITLE
Pretrained teacher model

### ISSUE THREAD

**Issue by gongshichina** [denguir/student-teacher-anomaly-detection#2]
Thanks for your codes! Could you please share the trained teacher model weights?

**Comment by yangninghua** [comment]
hi gongshichina, have you encountered training teacher_training.py loss is very large

**Comment by yangninghua** [comment]
Because the source code cannot run through, I made the following changes. Is this the cause of the extremely large loss?
```
def compactness_loss(output):
    # dim: (batch, vector)
    _, n = output.size()

    #avg = torch.mean(output, dim=1)
    #std = torch.std(output, dim=1)

    avg = torch.mean(output, dim=1)
    std = torch.std(output, dim=1)
    #zt = output.T - avg
    zt = output.t() - avg
    zt /= std
    #corr = torch.matmul(zt.T, zt) / (n - 1)
    corr = torch.matmul(zt.t(), zt) / (n - 1)
    loss = torch.sum(torch.triu(corr, diagonal=1)**2)
    return loss
```
